scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -370,6 +370,29 @@ void jobSubmitted(InternalJob job) {
         ClientJobState clientJobState = new ClientJobState(job);
         jobs.put(job.getId(), new JobData(job));
         listener.jobSubmitted(clientJobState);
+        if (job.getParentId() != null) {
+            // If a job has a parent, it means the parent job children count has been increased.
```

Comment:
If a job has a parent, it means **that** the parent job's children count has been increased

Reply:
ok